lib/datadog/tracing/trace_digest.rb

```diff
@@ -52,6 +52,10 @@ class TraceDigest
       # @!attribute [r] trace_service
```

Comment:
Can you make a note in the documentation of this file that we should change all references from `span` to `parent` in the future?